src/views/createEditCommunity/create.tsx

```diff
@@ -664,22 +701,65 @@ function CreateCommunityScreen() {
                         )}
                     </View>
                 </Modal>
+                <Modal
+                    visible={toggleLeaveFormModal}
+                    title={i18n.t('modalLeaveTitle')}
+                    onDismiss={() => setToggleLeaveFormModal(false)}
+                >
+                    <View
+                        style={{
+                            width: '100%',
+                            flexDirection: 'row',
+                            marginBottom: 16,
+                        }}
+                    >
+                        <Text
```

Comment:
We should try to avoid in line styling as much as possible in order to improve code readability and styles on stylesheets.